data/applications.yml

```diff
@@ -659,6 +659,28 @@
   dashboard_url: false
   description: 'GOV.UK Licensing (formerly ELMS, Licence Application Tool, & Licensify)'
 
+# Coronavirus
+
+- github_repo_name: govuk-coronavirus-vulnerable-people-form
+  private_repo: false
+  type: Specialist apps
+  production_hosted_on: paas
+  team: "#govuk-vulnerable-people-tech"
+  sentry_url: false # TODO - can the reviewer let me know what this should be?
+  deploy_url: 'https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/master/concourse/pipeline.yml'
+  dashboard_url: 'https://gds.splunkcloud.com/en-GB/app/gds-006-govuk/d006_coronavirus?form.main.earliest=-7d%40h&form.main.latest=now&form.xhost=coronavirus-vulnerable-people.service.gov.uk'
+  description: 'Helps citizens get support during the Covid-19 pandemic'
+
+- github_repo_name: govuk-coronavirus-business-volunteer-form
+  private_repo: false
+  type: Specialist apps
+  production_hosted_on: paas
+  team: "#govuk-corona-forms-tech"
+  sentry_url: false # TODO - can the reviewer let me know what this should be?
```

Comment:
```suggestion
  sentry_url: false https://sentry.io/organizations/govuk/issues/?project=5170680
```

Reply:
If this is just for reference then the above URL should be right,
otherwise if this is supposed to be the DNS might need removing here as that's a secret?

Reply:
I think that URL is right.